com.unity.perception/Tests/Editor/CharacterToolingTests.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Perception.Content;
+
+namespace CharacterToolingTests
+{
+    public class CharacterToolingTests
+    {
+        public List<GameObject> selectionLists = new List<GameObject>();
+        public List<string> assets = new List<string>();
+        private CharacterTooling contentTests = new CharacterTooling();
+
+        [SetUp]
+        public void Setup()
+        {
+            selectionLists = new List<GameObject>();
+            AssetListCreation();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            selectionLists.Clear();
+        }
+
```

Comment:
I think you are going to need tests where creating ears & nose fails (if that is possible to do) also tests for the validation, perhaps test against a valid model and also at least one test where the model does not pass validation

Reply:
Plan is next task will have tests added for character tool in that PR, I did add a few tests in this PR as well